src/libclipper/src/rpc_service.cpp

```diff
@@ -98,6 +104,7 @@ void RPCService::manage_service(const string address) {
   // Note that zeromq socket id is a byte vector
   log_info_formatted(LOGGING_TAG_RPC, "RPC thread started at address: ", address);
   boost::bimap<int, vector<uint8_t>> connections;
+  std::unordered_map<std::vector<uint8_t>, std::pair<VersionedModelId, int>, VectorHasher<uint8_t>> containers;
```

Comment:
Can you give `containers` a more descriptive variable name and add a comment about it's purpose? I got confused because I didn't look at the type signature closely enough and thought it was storing the actual `ModelContainer` objects. It's really a map from ZeroMQ connection ID to Clipper container ID (where a clipper container ID is the three part key `model_name, model_version, container_number`).